pkg/reconciler/openshift/tektonconfig/rbac.go

```diff
@@ -376,6 +378,17 @@ func (r *rbac) handleSCCInNamespace(ctx context.Context, ns *corev1.Namespace) e
 	return nil
 }
 
+func getRBACMaxCalls() int {
+	envValue := os.Getenv("OCP_RBAC_MAX_CONCURRENT_CALLS")
```

Comment:
@jkhelil keep this variable in https://github.com/tektoncd/operator/blob/main/pkg/reconciler/openshift/const.go